pkg/winapi/types.go

```diff
@@ -190,3 +193,91 @@ type DiskPerformance struct {
 	StorageDeviceNumber uint32
 	StorageManagerName  [8]uint16
 }
+
+type HInternet uintptr
+
+// Wrap it to provide useful String() method
+// syscall sets the inline struct's field as it was same-level field
+type Lpwstr struct {
```

Comment:
According to Go convention this type should be named LPWStr.

Reply:
this is the name of windows data type:
https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#lpwstr

Reply:
Yeah I know. I'm speaking about Go code :)

The correct names are:
- either LPWSTR (same as original Windows name). Golang Windows packages use this style for constant declarations but not types.
- either LPWStr (following Go [guidelines](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). The same notation style used for Windows types in Golang Windows packages e.g. `golang.org/x/sys/windows/types_windows.go` and in cagent `winapi` package)

Reply:
Yes, I got what you mean. Fixed